examples/contrib/se3transformer_qm9.py

```diff
@@ -0,0 +1,196 @@
+import argparse
+import os.path as osp
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from tqdm import tqdm
+
+import torch_geometric.transforms as T
+from torch_geometric.contrib.nn import SE3Transformer
+from torch_geometric.contrib.utils import Fiber
+from torch_geometric.data import Data
+from torch_geometric.datasets import QM9
+from torch_geometric.loader import DataLoader
+
+TASK_CHOICES = [
+    "mu",
+    "alpha",
+    "homo",
+    "lumo",
+    "gap",
+    "r2",
+    "zpve",
+    "U0",
+    "U",
+    "H",
+    "G",
+    "Cv",
+    "U0_atom",
+    "U_atom",
+    "H_atom",
+    "G_atom",
+    "A",
+    "B",
+    "C",
+]
+data_path = osp.join(osp.dirname(osp.realpath(__file__)), "data", "QM9")
+
+
+class ChooseTarget(T.BaseTransform):
+    def __init__(self, target: int) -> None:
+        self.target = target
+
+    def __call__(self, data: Data) -> Data:
+        data.y = data.y[:, self.target]
+        return data
+
+
+class AsFiber(T.BaseTransform):
+    def __init__(self, node_fiber: Fiber, edge_fiber: Fiber) -> None:
+        self.node_fiber = node_fiber
+        self.edge_fiber = edge_fiber
+
+    def __call__(self, data: Data) -> Data:
+        data.node_feats = self.node_fiber.tensor_to_dict(data.x)
+        data.edge_feats = self.edge_fiber.tensor_to_dict(data.edge_attr)
```

Comment:
Why rename the variables here. Keep them as `data.x` and `data.edge_attr`?

Reply:
Sure, I don't think there's any good reason to rename